### DISCUSSION TITLE
Administration Permissions

### DISCUSSION THREAD

**Discussion by 420skywalker420** [serenity-is/Serenity#5704] [Help] [answered]
Hi guys,

How to show specific columns only to users with administration permissions?

Thank you!

**Comment by edwardch** [comment, marked as answer]
Try using `[ReadPermission("Some permission")]` and/or `[FieldReadPermission("Some permission")]` on your field in YourRow.cs

**Reply by 420skywalker420** [reply]
Thank you, sir!
